hasher-matcher-actioner/src/OpenMediaMatch/storage/postgres/impl.py

```diff
@@ -525,12 +525,28 @@ def bank_delete(self, name: str) -> None:
     def bank_content_get(
         self, ids: t.Iterable[int]
     ) -> t.Sequence[interface.BankContentConfig]:
-        return [
-            b.as_storage_iface_cls()
-            for b in database.db.session.query(database.BankContent)
+        contents = list(
+            database.db.session.query(database.BankContent)
             .filter(database.BankContent.id.in_(ids))
             .all()
-        ]
+        )
+        return [b.as_storage_iface_cls() for b in contents]
```

Comment:
blocking: This change causes an extra O(n) of work, because you are copying it into a list, and then copying it again. 

Suggest leaving the prior version of the code